guides/front_end/vue_quick_start.mdx

```diff
@@ -0,0 +1,131 @@
+
+## 3. Add InstantSearch
+
+Include InstantSearch into `main.js` to include the Vue InstantSearch library.
+
+```js
+import { createApp } from 'vue';
+import App from './App.vue';
+import InstantSearch from 'vue-instantsearch/vue3/es';
+
+const app = createApp(App);
+app.use(InstantSearch);
+app.mount('#app');
+```
+
+## 4. Initialize the search client
+
+Use the following URL and API key to connect to a Meilisearch instance which contains data from Steam video games. Add the code below to the `App.vue` file.
+
+```js
+<template>
+  <ais-instant-search :search-client="searchClient" index-name="steam-video-games">
+  </ais-instant-search>
+</template>
+
+<script>
+import { instantMeiliSearch } from "@meilisearch/instant-meilisearch";
+
+export default {
+  data() {
+    return {
+      searchClient: instantMeiliSearch(
+        'https://ms-adf78ae33284-106.lon.meilisearch.io',
+        'a63da4928426f12639e19d62886f621130f3fa9ff3c7534c5d179f0f51c4f303',
+      ).searchClient,
+    };
+  },
+};
+</script>
+```
+
+ The `ais-instant-search` widget is the mandatory wrapper that allows you to configure your search. It takes two props: the `search-client` and the [`index-name`](/learn/core_concepts/indexes#index-uid).
+
+ ## 5. Add a search bar and list search results
+
+Add the `ais-search-box` and `ais-hits` widgets inside the `ais-instant-search` wrapper widget. 
+
+Import the CSS library to style the search components.
+
+```
+<template>
+  <ais-instant-search :search-client="searchClient" index-name="steam-video-games">
+  <ais-search-box />
+    <ais-hits>
+      <template v-slot:item="{ item }">
+	  <div>
+	    <img :src="item.image" align="left" :alt="item.name"/>
+          <h2>{{ item.name }}</h2>
+	    <p> {{ item.description }}</p>
+	  </div>
+      </template>
+    </ais-hits>
+  </ais-instant-search>
+</template>
+
+<script>
+import { instantMeiliSearch } from "@meilisearch/instant-meilisearch";
+import "instantsearch.css/themes/satellite-min.css";
+
+
+export default {
+data() {
+    return {
+    searchClient: instantMeiliSearch(
+        'https://ms-adf78ae33284-106.lon.meilisearch.io',
+        'a63da4928426f12639e19d62886f621130f3fa9ff3c7534c5d179f0f51c4f303',
+    ).searchClient,
+    };
+},
+};
+</script>
+```
+
+Use the slot directive to customize how each search result is rendered.
+
+<Capsule intent="tip">
+Use the following CSS classes to add custom styles to your components:
+`.ais-InstantSearch`, `.ais-SearchBox`, `.ais-InfiniteHits-list`, `.ais-InfiniteHits-item`
+</Capsule>
+
+## 6.Start the app and search as you type
+
+Start the app by running:
+
+```bash
+npm run dev
+```
+
+Now open your browser, navigate to your Vue app URL (e.g., `localhost:5173`), and start searching.
+
+![Vue app search UI with a search bar at the top and search results for four video games](/assets/images/react_quick_start/react-qs-search-ui.png)
```

Comment:
I made a similar comment in another PR. Whenever possible (which is not always), it's better to avoid mentioning unimportant info that might easily change with a new layout.

```suggestion
![Vue app search UI with a search bar at the top and search results a few video games](/assets/images/react_quick_start/react-qs-search-ui.png)
```